node/src/BaseClient.ts

```diff
@@ -348,20 +348,63 @@ function convertGlideRecordForZSet(
 
 /**
  * @internal
- * Downcast `GlideRecord` to `Record`. Use if you are 146% aware that `data` keys are always strings.
+ * Recursevly downcast `GlideRecord` to `Record`. Use if you are 146% aware that `data` keys are always strings.
  */
 export function glideRecordToRecord<T>(
```

Comment:
Just a minor suggestion, how about renaming this function to `castGlideRecordToRecord`?

Reply:
Please remind me to do this in #2207